# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../metamail/00_mm-2.7-make.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mm2.7/src/Makefile.orig	Wed Jan 26 12:32:33 1994
+++ mm2.7/src/Makefile	Wed Oct 11 21:44:12 1995
@@ -38,7 +38,7 @@
 
 STATICFLAG=FOOBAR${HOST_ARCH}
 
-CFLAGS = -g -I.  ${$(STATICFLAG)} 
+CFLAGS = $(RPM_OPT_FLAGS) -I. -DLINUX ${$(STATICFLAG)} 
 # The following is better if you want to make sure you run with SYSV defined
 # CFLAGS = -g -I.  ${$(STATICFLAG)}  -DSYSV
 # Also, for SGI Irix, compile in K&R mode
@@ -75,7 +75,7 @@
 INSTALL = install -c -s
 
 # Root of installation tree
-INSTROOT = /usr/local
+INSTROOT = /usr
 #
 # This is where binaries should be copied
 LOCALBINDIR = ${INSTROOT}/bin
@@ -91,7 +91,7 @@
 FONTSUFFIX=pcf
 
 # This is where the mailcap file should be put
-MAILCAPDIR = ${INSTROOT}/etc
+MAILCAPDIR = /etc
 
 # This helps with the "for" constructs below on some platforms:
 SHELL=/bin/sh
@@ -145,7 +145,7 @@
 	    echo "installing man page $$target";\
 	    ${CP} $$file $$target; 		\
 	done)
-	@echo "Renaming old mailcap file to $(MAILCAPDIR)/mailcap.old
+	@echo "Renaming old mailcap file to $(MAILCAPDIR)/mailcap.old"
 	-mv $(MAILCAPDIR)/mailcap $(MAILCAPDIR)/mailcap.old
 	@echo "installing file $(MAILCAPDIR)/mailcap"
 	${CP} mailcap $(MAILCAPDIR)
--- mm2.7/src/Makefile.don	Thu Feb 22 14:12:21 1996
+++ mm2.7/src/Makefile	Thu Feb 22 14:12:54 1996
@@ -145,10 +145,6 @@
 	    echo "installing man page $$target";\
 	    ${CP} $$file $$target; 		\
 	done)
-	@echo "Renaming old mailcap file to $(MAILCAPDIR)/mailcap.old"
-	-mv $(MAILCAPDIR)/mailcap $(MAILCAPDIR)/mailcap.old
-	@echo "installing file $(MAILCAPDIR)/mailcap"
-	${CP} mailcap $(MAILCAPDIR)
 
 uninstall-all:
 	@(cd bin; 				\
@@ -172,8 +168,6 @@
 	    echo "rm-ing man page $$target";\
 	    ${RM} $$target; 		\
 	done)
-	@echo "rm-ing file $(MAILCAPDIR)/mailcap"
-	@${RM} $(MAILCAPDIR)/mailcap
 
 clean:
 	-${RM} -rf *.BAK
--- mm2.7/src/Makefile.don	Thu Feb 22 14:19:05 1996
+++ mm2.7/src/Makefile	Thu Feb 22 14:19:27 1996
@@ -138,13 +138,6 @@
 	    echo "installing man page $$target";\
 	    ${CP} $$file $$target; 		\
 	done)
-	@(cd man;				\
-	for file in *.4;			\
-	do 					\
-	    target=$(MAN4DIR)/`basename $$file .4`.$(MAN4EXT); \
-	    echo "installing man page $$target";\
-	    ${CP} $$file $$target; 		\
-	done)
 
 uninstall-all:
 	@(cd bin; 				\
